ledger/streaming/api/src/error.rs

```diff
@@ -5,24 +5,27 @@ use grpcio::Error as GrpcError;
 use mc_api::ConversionError;
 
 #[derive(Debug, Display)]
-pub enum Error {
+pub enum LedgerStreamingError {
```

Comment:
I disagree with this rename. Callers can alias this when importing.

FWIW I modeled this after examples from our repo as well as other libs:
https://docs.rs/grpcio/latest/grpcio/enum.Error.html
https://github.com/mobilecoinfoundation/mobilecoin/blob/93e41f75df3f6fc202eaaa3099033bc845a92abe/ledger/db/src/error.rs#L9
https://github.com/mobilecoinfoundation/mobilecoin/search?q=enum.Error

Reply:
Okay cool, thanks for the references, I'll use it the original